src/main/java/org/antlr/intellij/plugin/ANTLRv4ASTFactory.java

```diff
@@ -16,10 +16,9 @@
 import org.antlr.intellij.plugin.psi.*;
 
 import java.util.HashMap;
-import java.util.Map;
 
 public class ANTLRv4ASTFactory extends ASTFactory {
-	private static final Map<IElementType, PsiElementFactory> ruleElementTypeToPsiFactory = new HashMap<>();
+	private static final HashMap<IElementType, PsiElementFactory> ruleElementTypeToPsiFactory = new HashMap<>();
```

Comment:
Do you have an explanation of why this improves performance? I've never heard of this before